<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameUserDialog</class>
 <widget class="QDialog" name="RenameUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rename User</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="form_layout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Full name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="firstNameLabel">
       <property name="text">
        <string>First name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="first_name_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lastNameLabel">
       <property name="text">
        <string>Last name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="last_name_edit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="fullNameLabel">
       <property name="text">
        <string>Display name:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="full_name_edit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="logonNameLabel">
       <property name="text">
        <string>Logon name:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="upn_prefix_edit"/>
       </item>
       <item>
        <widget class="QComboBox" name="upn_suffix_edit"/>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="logonNamePreWindows2000Label">
       <property name="text">
        <string>Logon name (pre-Windows 2000):</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="sam_name_domain_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sam_name_edit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name_edit</tabstop>
  <tabstop>first_name_edit</tabstop>
  <tabstop>last_name_edit</tabstop>
  <tabstop>full_name_edit</tabstop>
  <tabstop>upn_prefix_edit</tabstop>
  <tabstop>upn_suffix_edit</tabstop>
  <tabstop>sam_name_edit</tabstop>
  <tabstop>sam_name_domain_edit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>RenameUserDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>RenameUserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
